src/urbit/app/chess.hoon

```diff
@@ -676,51 +764,98 @@
 --
 |%
 ++  try-move
-  |=  [game=chess-game position=chess-position move=chess-move]
-  ^-  [new=(unit [game=chess-game position=chess-position]) cards=(list card)]
-  ?.  ?=(~ result.game)
+  |=  [game=active-game-state move=chess-move]
+  ^-  [new=(unit active-game-state) cards=(list card)]
+  ?.  ?=(~ result.game.game)
     [~ ~]
   =/  new-position
-    (~(apply-move with-position position) move)
+    (~(apply-move with-position position.game) move)
   ?~  new-position
     [~ ~]
-  =/  updated-game  `chess-game`game
+  =/  updated-game  `chess-game`game.game
```

Comment:
Would it be reasonable to change a face somewhere to avoid instances of `game.game`?

Reply:
Agreed, `game` in the input to `try-move` should be `game-state`